tools/schemacode/schemacode/render.py

```diff
@@ -119,6 +138,8 @@ def make_glossary(schema):
         obj_desc = obj_desc.replace("\n\n", "<br>")
         # Otherwise a newline corresponds to a space
         obj_desc = obj_desc.replace("\n", " ")
+        # Spec internal links need to be replaced
+        obj_desc = obj_desc.replace("SPEC_ROOT", get_relpath(src_path))
```

Comment:
won't this fail when `src_path` is `None`?

Reply:
Indeed. `posixpath.relpath(".", None)` works fine, but `posixpath.dirname(None)` doesn't and I forgot that I added that. Please see https://github.com/bids-standard/bids-specification/pull/1096/commits/c7bd1bcd5d5ea3c0db8314cafea38827fe900526.